File: Ritika/Arrrays/count_duplicates.cpp
#include <iostream>
using namespace std;

#define MAX_SIZE 100

int main()
{
    int arr[MAX_SIZE];
    int i, j, size, count = 0;
    cout<<"Enter size of the array : "<<endl;
    cin>>size;
    cout<<"Enter elements in array : "<<endl;
    for(i=0; i<size; i++)
    {
        cin>>arr[i];
    }

    for(i=0; i<size; i++)
    {
        for(j=i+1; j<size; j++)
        {
            if(arr[i] == arr[j])
            {
                count++;
                break;
            }
        }
    }

    cout<<"Total number of duplicate elements found in array = "<<count<<endl;

    return 0;
}

File: Arnav Sharma/Assignment-1/count_duplicates.java
package com.array;

public class count_duplicates {
    public static void main(String[] args) {

        int i = 0, j, count = 0;
        int[] arr = {10, 15, 25, 10, 8, 12, 10, 15, 55, 10, 60};

        while(i < arr.length)
        {
            j = i + 1;
            while(j < arr.length)
            {
                if(arr[i] == arr[j]) {
                    count++;
                    break;
                }
                j++;
            }
            i++;
        }
        System.out.println("\nThe Total Number of Duplicates  = " + count);
    }
}
